modules/ROOT/pages/alternatives.adoc

```diff
@@ -1,37 +1,21 @@
 = Setting alternatives
 
-Due to an https://github.com/fedora-sysv/chkconfig/issues/9[ongoing issue] in how alternatives configurations are stored on the system, Fedora CoreOS systems can not use the usual `alternatives` commands to configure them.
+Now Fedora CoreOS systems can use the usual `alternatives` commands to configure the default command, as https://github.com/fedora-sysv/chkconfig/issues/9[issue] is fixed.
 
-Instead, until this issue is resolved, you can set the symlinks directly in `/etc/alternatives`. For example, to use the legacy-based variants of the `iptables` commands:
+For example, to use the legacy-based variants of the `iptables` commands:
 
-[source,yaml,subs="attributes"]
+[source,bash,subs="attributes"]
```

Comment:
Let's have a Butane example first and then a manual example using shell commands.